lib/Serialiser/JsonLd.php

```diff
@@ -123,11 +123,15 @@ public function serialise(Graph $graph, $format, array $options = array())
         // OPTIONS
         $use_native_types = !(isset($options['expand_native_types']) and $options['expand_native_types'] == true);
         $should_compact = (isset($options['compact']) and $options['compact'] == true);
+        $should_expand = (isset($options['expand']) and $options['expand'] == true);
         $should_frame = isset($options['frame']);
 
         // expanded form
         $data = $ld_graph->toJsonLd($use_native_types);
 
+        if ($should_expand) {
+            $data = LD\JsonLD::expand($data);
+        }
```

Comment:
Current code uses independent if-clauses which check for compact or frame. You added a third one. 

As far as I understand the code, these options work as either-or and not x as well as y. Therefore I could override an `expand` by using the `frame` option.

* [ ] Can you add your if-clause to the existing code as `elseif`? So it would look like:

```php
if ($should_frame) {
    $data = LD\JsonLD::frame($data, $options['frame'], $options);
} elseif ($should_expand) {
    $data = LD\JsonLD::expand($data);
}
```

Refactoring the whole code section should not concern you and can be done in a separate PR.

Reply:
No problem, updated the PR.